deploy/helm/README.md

```diff
@@ -543,12 +548,12 @@ To enable, set `autoupdate.enabled: true`.
 
 ## Receive 3rd party metrics
 
-SWO K8s Collector has an OTEL service endpoint which is able to forward metrics and logs into SolarWinds Observability. All incoming data is properly associated with current cluster. Additionally, metrics are decorated with prefix `k8s.`.
+SWO K8s Collector has an OTEL service endpoint which is able to forward metrics, logs and traces into SolarWinds Observability. All incoming data is properly associated with current cluster. Additionally, metrics are decorated by default with prefix `k8s.` (prefix can be configured).
 
 Service endpoint is provided in format
 
 ```text
-"<chart-name>-metrics-collector.<namespace>.svc.cluster.local:4317"
+"<chart-name>-gateway.<namespace>.svc.cluster.local:4317"
```

Comment:
Since we are keeping the old service in this release, we could keep it here, too. Or mention that this new service is available since in the new collector.